.pipelines/templates/package-helm-chart.yaml

```diff
@@ -1,10 +1,20 @@
+parameters: 
+  - name: releaseTag
+    type: boolean
+    default: false
+
 steps: 
 - script: |
     echo "##vso[task.setvariable variable=UNIQUE_TAG]$(git rev-parse --short HEAD)"
     echo ${UNIQUE_TAG}
   displayName: Create Unique Tag
 - script: |
-    echo "##vso[task.setvariable variable=CHECKOUT_TAG]$(image.tag)-${UNIQUE_TAG}"
+    echo ${{ parameters.releaseTag }}
```

Comment:
What is the purpose of this change? Do you ever set releaseTag to false? Can we add a comment here explaining this

Reply:
The pr e2e will use the default false value of releaseTag, so the packaged chart name will just be osm-arc-<git-hash>. The release-e2e-job will pass in releaseTag as true when it calls this yaml, which adds an additional `-release` to the name of the packaged chart. Otherwise both the release-chart and pr-chart will have identical names.

Reply:
Update: I changed the package/publish steps so that publish is able to use the current version of OSM-arc and add a unique id to the image tag before pushing it to staging. For the release-job, we add "-release" to the image name, and for the "pr-job", we add "-pr" to the image name.